src/org/opendatakit/briefcase/ui/export/ExportPanelForm.java

```diff
@@ -163,14 +153,14 @@ public void refresh() {
     formsTable.refresh();
   }
 
-  public void showExporting() {
-    exportingLabel.setText(EXPORTING_DOT_ETC);
-    exportingLabel.setVisible(true);
+  public void showExportProgressBar() {
+    exportProgressBar.setVisible(true);
   }
 
-  public void hideExporting() {
-    exportingLabel.setVisible(false);
-    exportingLabel.setText(EXPORTING_DOT_ETC);
+  public void hideExportProgressBar() {
+    exportProgressBar.setValue(100);
```

Comment:
Maybe just `setVisible(false)`? I believe setting the value and the indeterminate properties won't have any effect here...